### DISCUSSION TITLE
Doc outdated on Cart ? Non-static method 'use' for CartSessionManager

### DISCUSSION THREAD

**Discussion by SebastianBotez** [lunarphp/lunar#1502] [Q&A] [answered]
Hello guys.
I have a question regarding the documentation.
I'm at the part where I need to add items to cart and I try to retrieve the current cart.
If the current cart returns NULL, I create a new one and store it in the session (at least I try).

```
$cart = \Lunar\Facades\CartSession::current();
if(!$cart) {
    $cart = \Lunar\Models\Cart::create([
        'currency_id' => 1,
        'channel_id' => 1,
    ]);
    CartSessionManager::use($cart);
}
```

Unfortunatelly I'm getting
`Non-static method Lunar\Managers\CartSessionManager::use() cannot be called statically` wich makes sense because the `use` function is declared as non-static `public function use(Cart $cart)`

Official documentation says as below wich is equivalent with my code.

```
$cart = \Lunar\Models\Cart::first();
CartSessionManager::use($cart);
```

Thank for all your effort!

**Comment by glennjacobs** [comment]
If you are getting `cannot be called statically` it suggests you are not using the Facade.

**Reply by SebastianBotez** [reply]
Yes indeed, but unfortunatelly there is no Facade defined as CartSessionManager or maybe I could not find it. Am i wrong?

<img width="637" alt="Screenshot 2024-01-29 at 13 38 50" src="https://github.com/lunarphp/lunar/assets/10911720/5614d2e6-330f-4874-8200-7809dd9615af">

**Reply by glennjacobs** [reply, marked as answer]
\Lunar\Facades\CartSession
